jekyll/_cci2/docker-layer-caching.md

```diff
@@ -47,7 +47,7 @@ jobs:
   build:
     docker:
       # DLC does nothing here, its caching depends on commonality of the image layers.
-      - image: circleci/node:14.17.3-buster-browsers
+      - image: cimg/node:14.17.3-buster-browsers
```

Comment:
as we stopped supporting `buster` variant, I believe our mapping here would be:

```suggestion
      - image: cimg/node:14.17.3-browsers
```
based on https://hub.docker.com/r/cimg/node/tags?page=1&name=14.17.3